packages/golub/example/app/android/app/src/main/java/io/flutter/plugins/Messages.java

```diff
@@ -314,7 +314,7 @@ static void setUp(
         BasicMessageChannel<Object> channel =
             new BasicMessageChannel<>(
                 binaryMessenger,
-                "dev.flutter.pigeon.pigeon_example_package.ExampleHostApi.getHostLanguage"
+                "dev.flutter.pigeon.golub_example_package.ExampleHostApi.getHostLanguage"
```

Comment:
let's discuss replacement for `dev.flutter.pigeon` package name, maybe `dev.yobari.golub` could be our choice, but again – swear word in package name(

Reply:
done